src/config/serverConfigDefaults.ts

```diff
@@ -45,6 +45,8 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     skin_description:
         'The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets',
     show_genomenexus: true,
+    skin_study_view_blurb:
```

Comment:
This default will be removed before merging. The value should come from the server which https://github.com/cBioPortal/cbioportal/pull/7718 is trying to address that.